www/index.html

```diff
@@ -66,15 +66,30 @@
     <style is="custom-style" include="shared-styles"></style>
 
     <style is="custom-style">
+
+
+        html, body {
+            height: 100%;
+            margin: 0;
+        }
+
+        paper-scroll-header-panel {
+            height: 100%;
+        }
+
         paper-toolbar.custom {
             --paper-toolbar-background: var(--paper-green-700);
+            --paper-toolbar-title: {
+                font-weight: bold;
+            };
         }
 
         paper-spinner.multi {
-            position: absolute;
-            margin-left: 50%;
-            margin-top: 50%;
-            z-index: 1000;
+            /*position: absolute;*/
+            /*margin-left: 50%;*/
+            /*margin-top: 50%;*/
+            display:block;
+            /*z-index: 1000;*/
```

Comment:
Fixed in https://github.com/UnderDevCrew/HapplyHour/pull/5/commits/0725585da29f34c37c9e14fc1f911c0f3f86823e